src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -270,6 +270,7 @@ private boolean areQuestionNumbersConsistent(List<FeedbackQuestionAttributes> qu
     /**
      * Returns true if a session has question in either STUDENTS type or TEAMS type.
      */
+    @Deprecated
     public boolean sessionHasQuestions(String feedbackSessionName, String courseId) {
         return fqDb.hasFeedbackQuestionsForGiverType(feedbackSessionName, courseId, FeedbackParticipantType.STUDENTS)
                 || fqDb.hasFeedbackQuestionsForGiverType(feedbackSessionName, courseId, FeedbackParticipantType.TEAMS);
```

Comment:
Can just remove this altogether since it is not used anywhere after this change.